blacklist_service/src/main/resources/application.yml

```
@@ -0,0 +1,20 @@
+grpc:
+  server:
+    port: 6566
+server:
+  port: 6567
+spring:
```

давайте уберем ебучий баннер 
```suggestion
spring:
  main:
    banner-mode: off
```